### PR TITLE
fix: add new var + remove duplicate + reorg alphabetically

### PR DESCRIPTION
**Pull request by openroadie**
Added documentation for new flow variable.

### CODE REVIEW

File: docs/user/FlowVariables.md [hunk 1]
```diff
@@ -162,10 +163,9 @@ Note:
 |-----------------------|--------------------------------------------------------------------------------------------------------------|
 | `CTS_BUF_CELL`        | The buffer cell used in the clock tree.                                                                      |
 | `FILL_CELLS`          | Fill cells are used to fill empty sites.    								       |
-| `SETUP_SLACK_MARGIN`  | Specifies a time margin for the slack when fixing setup violations.                                          |
 | `HOLD_SLACK_MARGIN`   | Specifies a time margin for the slack when fixing hold violations. This option allow you to overfix.         |
-| `FILL_CELLS`          | Fill cells are used to fill empty sites.								       |
-
+| `SETUP_SLACK_MARGIN`  | Specifies a time margin for the slack when fixing setup violations.                                          |
+| 'TNS_END_PERCENT'     | Specifies how many percent of violating paths to fix [0-100]. Worst path will always be fixed                |
```
Comment (vijayank88) [inline docs/user/FlowVariables.md]:
```suggestion
| `TNS_END_PERCENT`     | Specifies how many percent of violating paths to fix [0-100]. Worst path will always be fixed                |

```

**Review by vijayank88 — CHANGES_REQUESTED** [review]

**Review by openroadie — COMMENTED** [review]
Sorry... still figuring out git. Requested changes are made.

**Comment by vijayank88** [comment]
@openroadie 
Is there default value available for `TNS_END_PERCENT`?

**Comment by vijayank88** [comment]
Looks good. Leave it to Matt for approval.

**Comment by vijayank88** [comment]
@openroadie 
It's nice to add `export TNS_END_PERCENT ?= 0` in all `flow/platforms/.../config.mk`.
Update in description as follows:
```
Specifies how many percent of violating endpoints to fix [0-100]. Worst endpoints will be fixed always. Default is `0`
```

**Review by maliberty — APPROVED** [review]

**Pull request merged**